select/sdks/ios/guide-v2.md

```diff
@@ -62,13 +62,13 @@ pod update Fidel
 
 > Info: If you prefer to use Cocoapods, please check our instructions about integrating the SDK using Cocoapods, above.
 
-- Download the latest version of the `Fidel.xcframework` artefact from the `master` branch of our [GitHub repo](https://github.com/FidelLimited/fidel-ios).
+- Download the latest version of the `Fidel.xcframework` and `analytics.xcframework` artifacts from the `master` branch of our [GitHub repo](https://github.com/FidelLimited/fidel-ios).
```

Comment:
I don't understand why our client would want to download the analytics framework. I thought that was only a dependency of our web-sdk. What am I missing? 🤔

Reply:
**Context**

We always provided a way to integrate our mobile SDKs manually, if our customer does not want to use a dependency manager (like Cocoapods on iOS). We had customers who actually don’t use Cocoapods. An alternative would be to add the SDK manually, although it would be harder for the customer to maintain.

**Response**

> I don't understand why our client would want to download the analytics framework and install it independently. 

A customer does not have the option to decide to add analytics to their project or not. If they don’t add one of the artifacts, the iOS SDK does not compile. They have to add both artifacts to their project. But they have 2 options to integrate our iOS SDK:
1. Use Cocoapods as a dependency manager for adding all the artifacts of the iOS SDK (they would not care which, how many etc.). They would only manage the version of the SDK.
2. Use a manual integration, in which case they have to manually add all the iOS SDKs artifacts manually into their project.

I don’t think we have many (if any) customers who want to integrate the iOS SDK manually. If they do, they have to add the `analytics.xcframework` artifact manually, just like they have to add the Fidel.xcframework artifact. It is a piece of our SDK, just like it is for the web SDK and just like it would have been, if we would have went with [solution 3.2](https://fidelapi.atlassian.net/wiki/spaces/CL1/pages/2648998027/Segment+data+tracking+services+integration+into+the+mobile+SDKs#3.2.-Add-an-Analytics-module-inside-the-Fidel-SDK-to-work-with-the-Segment-API). In this case, we chose solution 3.1 and the artifact is separate because we built this module using Kotlin Multi-platform.

> I thought that was only a dependency of our web-sdk

I think you meant “our iOS SDK”. And the analytics library is a dependency of the iOS SDK. However, when integrating manually, you have to manage dependencies manually as well. Since a customer might have chosen manual integration approach, they would install the dependencies’ artifacts manually as well.

> What am I missing? 🤔

Hope my answer clarified things. Please let me know, if I did a bad/good job or if there is anything I can shed light on.